docs/en/commands/status.md

```diff
@@ -0,0 +1,75 @@
+# Status
+
+Retrieve the server’s current status.
+
+## Example Usage
```

Comment:
we should add examples in all languages. if we don't know what they look like, we should add a "TODO". but all should show up, I think.